include/cuco/detail/bloom_filter/bloom_filter_impl.cuh

```diff
@@ -97,7 +97,7 @@ class bloom_filter_impl {
   template <class CG>
   __device__ constexpr void clear(CG const& group)
   {
-    for (int i = group.thread_rank(); num_blocks_ * words_per_block; i += group.size()) {
+    for (int i = group.thread_rank(); i < num_blocks_ * words_per_block; i += group.size()) {
```

Comment:
@sleeepyjack Do we want to keep this CG clear? It doesn't seem to be used.

Reply:
Yes, we wAnt to keep this as this is the only way of initializing a shmem filter.